templates/firm/show.json.twig

```diff
@@ -1,23 +1,23 @@
 {
-    "@context": "https://schema.org",
+    "@context": "http://schema.org",
+    "@type": "Organization"
```

Comment:
I quite did not understand why do we need to take the organization out of the graph (I think this one also works but I am unsure what was  wrong with the previous version)

Reply:
I moved the organization out of the graph and into the root so that the JSON validated.